Add workshop on test selection in June 2025

salsolatragus opened the pull request:
Our online workshop on how to automatically select the most relevant test, to cut down test runtimes and speed up test feedback, even with long-running tests, higher-level tests and manual tests.

ckenst commented on _data/current.yml:
```diff
@@ -302,3 +302,9 @@
+
+- name: "Workshop: Everyone wants to shift left, but our tests are just too slow!"
+  location: Online
+  dates: "June 25, 2025"
```
Please put this workshop in the right place in the list. Move it up so it is in order by date. Thanks

salsolatragus commented:
> @salsolatragus are you able to fix this?

@ckenst Sorry, somehow I missed the notification about your comment! Fixed the order and rebased on the current master. Hope that solves the issue.

ckenst commented:
@salsolatragus thank you!
